### DISCUSSION TITLE
Add dist folder for CDN

### DISCUSSION THREAD

**Discussion by paribaker** [thinknimble/tn-models-fp#171] [General]
Add the dist folder so that we can serve the package via a CDN

**Comment by lakardion** [comment]
No one does this so I'm wondering whether this is actually the way to do it. 
I deliberately set the dist folder not to be tracked because that's what package maintainers do.

If you want to have the package served locally you can do start tracking the dist folder and reference it but I don't think we should keep the dist folder in main.

If you want to do something locally there's this: https://github.com/thinknimble/tn-models-fp?tab=readme-ov-file#side-by-side-debugging

**Comment by lakardion** [comment]
As per our private conversation. 
We will not commit the dist folder. Instead if anyone would like to use the cdn approach for the library. We would recommend: https://esm.sh

ig: https://esm.sh/@thinknimble/tn-models@4.0.0
